components/eamxx/src/share/io/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ set_target_properties(scream_io PROPERTIES
   Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/modules
 )
 target_include_directories(scream_io PUBLIC ${CMAKE_CURRENT_BINARY_DIR}/modules)
+
+if (DEFINED ENV{ADIOS2_DIR})
```

Comment:
Does scorpio handle this in the same way, namely that ADIOS support is enabled if the env var ADIOS2_DIR is defined at config time? I just want to make sure we don't run into the case where scorpio thinks it's using adios, but eamxx did not see the env var, so doesn't add the include dir.

Reply:
Yeah, SCORPIO also requires this env var set to configure with ADIOS